pkg/machines/components/create-vm-dialog/createVmDialog.jsx

```diff
@@ -113,6 +113,10 @@ function validateParams(vmParams) {
         validationFailed['source'] = _("Installation Source should not be empty");
     }
 
+    if (vmParams.memorySize === 0) {
+        validationFailed['memory'] = _("Memory should be more than 0");
```

Comment:
"more than 0" sounds a bit strange IMHO, and "should" is too weak -- a VM can't function without memory. Perhaps "Memory must not be 0"? @andreasn, any opinion?